.vscode/launch.json

```diff
@@ -131,6 +131,15 @@
       "args": ["build"],
       "cwd": "${workspaceRoot}/examples/basic"
     },
+    {
+      "name": "Generators",
+      "type": "lldb",
+      "request": "launch",
+      "preLaunchTask": "prepare turbo",
+      "program": "${workspaceRoot}/target/debug/turbo",
+      "args": ["gen", "blog - release post", "--args", "1.11.0", "1.10.0", "_", "tagline about my really cool release"],
```

Comment:
Do we want this to be a test?

Reply:
Yes will add in a follow up, I wanted to leave this in so we have a prebuild debug configuration for gen as well.